src/pipert2/core/managers/event_board.py

```diff
@@ -69,10 +70,11 @@ def get_event_notifier(self) -> Callable:
 
         """
 
-        def notify_event(event_name, output_event_queue, **kwargs):
-            output_event_queue.put(Method(event_name, kwargs))
+        def notify_event(event_name, output_event_queue, flow_name=None, routine_name=None, **kwargs):
```

Comment:
I know its not your code but please change the parameter name `kwargs` to params or something like that.